docs/observability/aws/deploy-use-aws-observability/view-dashboards.md

```diff
@@ -12,7 +12,7 @@ This feature is available in the following account plans.
 | Solution Component | CloudFlex | Credits |
 |:--|:--|:--|
 | AWS Observability Dashboards | Trial, Professional, and Enterprise | Trial, Essentials, Enterprise Operations, Enterprise Security,  Enterprise Suite |
-| Root Cause Explorer | Trail, Enterprise | Trial, Enterprise Operations, Enterprise Suite |
+| Root Cause Explorer | Trial, Enterprise | Trial, Enterprise Operations, Enterprise Suite |
```

Comment:
I think this should also now contain `Enterprise Suite - Flex` corresponding to Credits account plans